neural_modelling/src/neuron/structural_plasticity/synaptogenesis/topographic_map_impl.c

```diff
@@ -601,7 +601,7 @@ bool synaptogenesis_dynamics_formation_rule(void)
     uint no_elems = number_of_connections_in_row(
         synapse_row_fixed_region(rewiring_dma_buffer.row));
     if (no_elems >= rewiring_data.s_max) {
-        log_error("row is full");
+        log_debug("row is full");
         return false;
```

Comment:
The rest of this is fine - I was just wondering whether this change is related to Peter's branch(es), or do you want it included here?

Reply:
It is related to Peters branch so doesn't need be included, however, it was necessary to running properly. Peter seemed fine with it being changed too as it had little impact and did not need to error out anyway.

Reply:
Ok, in that case we'll keep it in here.